pkgs/development/python-modules/ansible-lint/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib
+, fetchFromGitHub
+, buildPythonPackage
+, ansible
+, pyyaml
+, six
+, nose
+, setuptools_scm
+, ruamel_yaml
+}:
+
+buildPythonPackage rec {
+  pname = "ansible-lint";
+  version = "4.1.0";
+
+  src = fetchFromGitHub {
```

Comment:
Why not `fetchPypi`? Please add a comment.

Reply:
fixed.